packages/riipen-ui/src/components/RadioButton.test.jsx

```diff
@@ -0,0 +1,154 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import { faFlag } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+import RadioButton from "./RadioButton";
+
+const icon = i => <FontAwesomeIcon icon={i} />;
+
+describe("<RadioButton>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(<RadioButton />);
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(<RadioButton />);
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("checked prop", () => {
+    it("sets class name as checked and color as initial when checked is true", () => {
+      const checked = true;
+
+      const wrapper = mount(<RadioButton checked={checked} readOnly />);
+
+      expect(
+        wrapper
+          .find("RadioButton")
+          .childAt(0)
+          .hasClass("checked")
+      ).toEqual(true);
+      expect(wrapper.find("Typography").props().color).toEqual("initial");
+    });
```

Comment:
checked is also used in the label and input elements. Maybe we should test those too?

Reply:
It's also used in `<input>` so we should probably add a test for that as well.

Reply:
Adjusted, let me know what you think of the test name.